src/compiler/crystal/tools/init.cr

```diff
@@ -56,6 +56,25 @@ DIR  - directory where project will be generated,
       github_user.empty? ? default : github_user
     end
 
+    def self.fetch_name(opts, args)
+      name = fetch_required_parameter(opts, args, "NAME")
+      name_type = fetch_name_type(name)
+      if name_type
+        puts "invalid NAME value: #{name} #{name_type} already exists"
```

Comment:
I hadn't thought too much about this message. What would you suggest to be a good error message? 

Secondly for printing as an error, I'll change it to `STDERR.puts`.